### PR TITLE
switch to colored logo

### LINKED ISSUE #838: Use green svg for logo (coordinate with Rob W)

### PR DESCRIPTION
**Pull request by na3d**
Closes #838 
screen shot 2017-01-03 at 4 48 55 pm

### CODE REVIEW

File: scss/components/Header.scss [hunk 1]
```diff
@@ -47,7 +47,7 @@ $header-padding: $grid-gutter-width-base / 2;
 
     &-logo {
         font-size: 35px; /* Special font-size case */
-        line-height: $main-nav-height;
+        line-height: 50px;
```
Comment (eatonphil) [inline scss/components/Header.scss]: Why the change here?

Comment (na3d) [inline scss/components/Header.scss]: because the logo wasn't aligned vertically

Comment (na3d) [inline scss/components/Header.scss]: I tried to resize the image, and other solutions, but that ended up being the one that worked the best.
